testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-all/ignite.properties

```diff
@@ -0,0 +1,4 @@
+connector.name=ignite
```

Comment:
Why do you add this file if it is invalid?

Reply:
Hi, I think here is for product test that test load all select connectors, the server in connection url is not actually be used.